WordPressKit/AccountServiceRemoteREST+SocialService.swift

```diff
@@ -23,7 +23,7 @@ extension AccountServiceRemoteREST {
                                        oAuthClientID: String,
                                        oAuthClientSecret: String,
                                        success: @escaping (() -> Void),
-                                       failure: @escaping ((NSError) -> Void)) {
+                                       failure: @escaping ((Error) -> Void)) {
```

Comment:
I don't think this counts as a breaking change, does it?

The API became looser not stricter. Any code passing `(NSError) -> Void` to failure should still compile now that the expected type is `(Error) -> Void`.

_Anyways, if it's a breaking change, I'll just add it to the existing list_ 😅

Reply:
IMO, it's still a breaking change. The consumers may use `NSError.domain` in their code, which won't compile with this updated code.

One way to avoid this breaking change is casing as NSError in the `failure` callback below: `failure(error as NSError)`.

Reply:
If that's okay with you, I'd keep `Error` and count this as a breaking change, then.

`Error` is more Swifty, which I think is the best approach going forward.

I'll merge under that assumption and the approval you already gave. Happy to followup with a fix PR if needed.

Reply:
👍 I don't have a strong opinion on this.